api/build.go

```diff
@@ -539,7 +549,33 @@ func GetBuilds(c *gin.Context) {
 	// nolint: gomnd // ignore magic number
 	perPage = util.MaxInt(1, util.MinInt(100, perPage))
 
-	b, t, err = database.FromContext(c).GetRepoBuildList(r, filters, page, perPage)
+	// capture before query parameter if present, default to now
+	//
+	// nolint: gomnd, lll // ignore magic number and long line length
+	before, err := strconv.ParseInt(c.DefaultQuery("before", strconv.FormatInt(time.Now().UTC().Unix(), 10)), 10, 64)
```

Comment:
question - what are the last two arguments?

Reply:
Thats the base (`10`) and the size in bits (`64`):

https://pkg.go.dev/strconv#ParseInt